COPYRIGHT

```diff
@@ -6,16 +6,17 @@ terms.
 
 Longer version:
 
-The Rust Project is copyright 2010, The Rust Project
-Developers.
+Copyrights in the Rust project are retained by their contributors. No
+copyright assignment is required to contribute to the Rust project.
```

Comment:
That's typically implicit when contributing to an Open Source project. Most projects don't state that explicitly. And I would suggest that we avoid explicitly writing language like "by contributing ... you agree ...", because that starts to look more like a contract or CLA.